studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -146,6 +155,52 @@ const OrganizationProjects = ({
           )}
         </ul>
       )}
+
+      <Modal
+        closable
+        hideFooter
+        size="small"
+        visible={orgBillingMigrationModalVisible}
+        onCancel={() => setOrgBillingMigrationModalVisible(false)}
+        header="Legacy-Billing"
```

Comment:
?

Reply:
?

Reply:
oh i see, its the title string.

Maybe be more descriptive, 'Legacy-billing' is the first thing someone is going to see.

"We're upgrading our billing" might be more appropriate